modules/zone/variables.tf

```
@@ -8,3 +8,11 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "vpc" {
+  description = "VPC ID"
+  type = object({
+    vpc_id = string
+  })
+  default = null
+}
```

Making this a list since the resource supports multiple vpc blocks...

```suggestion
variable "vpcs" {
  description = "List of objects of VPC IDs associate to the Private Hosted Zone. NOTE: At least one VPC object is required to create a Private Hosted Zone"
  type = list(object({
    vpc_id = string
  }))
  default = []
}
```